### DISCUSSION TITLE
图片识别报错

### DISCUSSION THREAD

**Discussion by 5Breeze** [STranslate/STranslate#103] [Q&A] [answered]
![image](https://github.com/ZGGSONG/STranslate/assets/89394068/5ddd1d19-f9ae-43b9-adfd-fcc7a475c498)
![image](https://github.com/ZGGSONG/STranslate/assets/89394068/5a683cf2-5628-4bb7-b6cb-fbda8540c23d)
an'zhuang'd安装的是这个版本

**Comment by ZGGSONG** [comment, marked as answer]
![image](https://github.com/ZGGSONG/STranslate/assets/49741009/b4fd3f4e-ef7b-46f8-ac52-131b54c32c0e)

跳转至发布页查看详情及演示 [1.1.4.523](https://github.com/ZGGSONG/STranslate/releases/tag/1.1.4.523)

1.1.4.523 开始分离了PaddleOCR依赖及数据包部分，全新安装需要到软件设置中点击下载即可自动加载数据。

**Reply by GeniusAng** [reply]
显示下载时发生异常怎么办

**Reply by ZGGSONG** [reply]
网络问题，可以去123网盘下载好后手动将压缩包放到软件根目录
方法1：打开软件设置-ocr-点击下载会自动解压
方法2：直接解压即可(记得不能再套目录，压缩包里的文件全部解压到根目录)
